core/Migrations/Version23000Date20210930122352.php

```diff
@@ -60,7 +60,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 				'notnull' => true,
 			]);
 			$table->setPrimaryKey(['id']);
-			$table->addUniqueIndex(['user_id'], 'user_id');
+			$table->addUniqueIndex(['user_id'], 'pc_user_id');
```

Comment:
```suggestion
			$table->addUniqueIndex(['user_id'], 'profile_config_user_id_idx');
```
Would suggest something more explicit :)